app/integrations/aws/sso_admin.py

```diff
@@ -0,0 +1,117 @@
+import os
+import logging
+from integrations.aws.client import execute_aws_api_call, handle_aws_api_errors
+
+
+logger = logging.getLogger(__name__)
+
+ROLE_ARN = os.environ.get("AWS_ORG_ACCOUNT_ROLE_ARN", "")
+INSTANCE_ARN = os.environ.get("AWS_SSO_INSTANCE_ARN", "")
+SYSTEM_ADMIN_PERMISSIONS = os.environ.get("AWS_SSO_SYSTEM_ADMIN_PERMISSIONS")
+VIEW_ONLY_PERMISSIONS = os.environ.get("AWS_SSO_VIEW_ONLY_PERMISSIONS")
+
+
+def get_predefined_permission_sets(permission_set_name):
+    """Get the predefined permission sets for AWS SSO.
+
+    Args:
+        permission_set_name (str): The name of the predefined permission set.
+
+    Returns:
+        str: The ARN of the predefined permission set.
+    """
+    predefined_permission_sets = {
+        "write": SYSTEM_ADMIN_PERMISSIONS,
+        "read": VIEW_ONLY_PERMISSIONS,
```

Comment:
What would you think of offering the predefined permission set names as constants to make it easier for calling processes to request the expected one?

Reply:
I'm open to anything really, I just wanted to simplify the handling of those based on the previous behaviour. There was simply a condition between permission == "write" and a toggle between Admin and Read permissions sets.

This way I was simply making it possible to either have a valid name passed through or a permission set ARN.

Reply:
Yeah, I thought that was interesting how you allow for a predefined permission set name or fallback to a passed ARN.  I guess my take would be if there going to be a lot of cases where callers are hardcoding `write` or `read` then I'd go for constants, otherwise, no need.

It's non-blocking, so I've approved since we can always add these constants later if they're needed!